README.md

```diff
@@ -46,6 +56,11 @@ all components of the genesis file and scripts to recompute it.
 Anyone with an ATOM allocation in the [`penultimate_genesis.json`](./penultimate_genesis.json) who intends to participate in the genesis ceremony must submit a pull request
 containing a valid `gen-tx` to this repository in the `/gentx` folder with a file name like `<moniker>.json`.
 
+It must be noted that it remains in the community's discretion to adopt or not
+to adopt the Genesis State that the ICF recommends within the Genesis Block
+Software. Therefore, the ICF cannot guarantee that (i) ATOMs will be created and
+(ii) the recommended allocation as set forth herein will actually take place
+
```

Comment:
This seems like a repeat of the following section in the Disclaimer?

Reply:
Purposeful repetition :)

Reply:
Ok I eased it.